src/python/sortingview/gui/extensions/mountainview/MVSortingComparisonView/MVSortingComparisonView.tsx

```diff
@@ -0,0 +1,239 @@
+import { faPencilAlt } from '@fortawesome/free-solid-svg-icons'
```

Comment:
This is the sorting comparison view window. It is modeled after MVSortingView. The difference is that it takes as input two sortings (instead of one) for comparing.